Userland/Libraries/LibJS/Tests/syntax/async-await.js

```diff
@@ -167,6 +168,16 @@ describe("non async function declaration usage of async still works", () => {
         const evalResult = eval("async >= 2");
         expect(evalResult).toBeTrue();
     });
+
+    test("async with line ending does not create a function", () => {
+        expect(() => {
+            async;
+            function f() {}
```

Comment:
This was probably "fixed" by prettier, should not have the semicolon

Reply:
Added prettier ignore with a comment